c2pa/c2pa_api/c2pa_api.py

```diff
@@ -193,7 +192,36 @@ def __init__(self, callback):
 def create_signer(callback, alg, certs, timestamp_url=None):
     return api.CallbackSigner(SignerCallback(callback), alg, certs, timestamp_url)
 
-
+# Because we "share" SigningAlg enum, seems we need to manually
+# coerce the enum types, as unffi itself does too
+def convert_to_alg(alg):
+    match str(alg):
+        case "SigningAlg.ES256":
+            return api.SigningAlg.ES256
+        case "SigningAlg.ES384":
+            return api.SigningAlg.ES384
+        case "SigningAlg.ES512":
+            return api.SigningAlg.ES512
+        case "SigningAlg.PS256":
+            return api.SigningAlg.PS256
+        case "SigningAlg.PS384":
+            return api.SigningAlg.PS384
+        case "SigningAlg.PS512":
+            return api.SigningAlg.PS512
+        case "SigningAlg.ED25519":
+            return api.SigningAlg.ED25519
+        case _:
+            raise ValueError("Unsupported signing algorithm: " + str(alg))
+
+def create_remote_signer(callback, alg=None):
+    alg_to_use = alg
+    if alg_to_use is not None:
```

Comment:
Would we even want that? The option to specify an algorithm? It's nice, but a little bit "use at your own risk"